recipes/python-libsbml/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: python-libsbml
+  version: "5.18.3"
+
+source:
+  url: https://github.com/sbmlteam/python-libsbml/releases/download/v5.18.3/python-libsbml.tar.gz
```

Comment:
Please template this do the bot can do auto-updates: `https://github.com/sbmlteam/python-libsbml/releases/download/v{{ version }}/python-libsbml.tar.gz`